api/package-lock.json

```diff
@@ -0,0 +1,14460 @@
+{
```

Comment:
Wrong package manager; Please delete package-lock.json and use `yarn` (with `yarn install`) instead.